config/testSetup.js

```diff
@@ -3,3 +3,22 @@ import Enzyme from 'enzyme'
 import Adapter from 'enzyme-adapter-react-16'
 
 Enzyme.configure({ adapter: new Adapter() })
+
+const INVALID_ARG_0 =
+    'Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: %s.%s'
+const INVALID_ARG_1 = 'undefined'
+const INVALID_ARG_2 =
+    " You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports."
+
+const originalConsoleError = global.console.error
+
+global.console.error = function () {
+    const isMessageToIgnore =
+        arguments[0] === INVALID_ARG_0 &&
+        arguments[1] === INVALID_ARG_1 &&
+        arguments[2] === INVALID_ARG_2
+
+    if (!isMessageToIgnore) {
+        originalConsoleError.apply(null, arguments)
+    }
+}
```

Comment:
This works as expected, but I'm not actually sure we want to add this. Ideally I would like to find a way to actually prevent these errors from showing in the log in the first place.

If we can't figure out how to do that, then perhaps we do want to keep this, but only when another condition is satisfied? For example:
```process.env.SUPPRESS_MISSING_COMPONENT_ERROR_LOG === true```

I don't like all this clutter to show during the test run, but I also worry that one day we will actually have a bad import/export somewhere and we don't see identify the problem due to this "fix"....

Reply:
OK, found an actual solution to the problem in 4f97da5e99c87d63536f8c5d4d3f3c703179e8af